elixir/apps/web/test/web/live/policies/new_test.exs

```diff
@@ -95,7 +95,7 @@ defmodule Web.Live.Policies.NewTest do
     html = render(form)
 
     disabled_input = Floki.find(html, "input[name='policy[actor_group_id]_name']")
-    assert Floki.attribute(disabled_input, "disabled") == ["disabled"]
+    assert Floki.attribute(disabled_input, "disabled") == [""]
```

Comment:
The new test helper html renderer sets `disabled=""` for disabled HTML elements, I double-checked this in an actual browser and it still works.